src/Exception/AwsException.php

```diff
@@ -81,6 +85,16 @@ public function getCommand()
     }
 
     /**
+     * Get the concise error message if any.
+     *
+     * @return string|null
+     */
+    public function getErrorMessage()
```

Comment:
I think this is too close in spirit to the `getMessage` method this class is inheriting from `\Exception`. A name like `getServiceMessage` (or something similar) would be better, as it would call out why this is different from the regular exception message and why it may or may not be present on an instance of `AwsException`.